grpclog/loggerv2_test.go

```diff
@@ -61,3 +66,231 @@ func checkLogForSeverity(s int, b []byte) error {
 	}
 	return nil
 }
+
+func TestLoggerV2PrintFunc(t *testing.T) {
```

Comment:
Can you convert the tests to table driven format like here [grpc-go/clientconn_test.go](https://github.com/grpc/grpc-go/blob/e6b6318ad90847fa8b5ed987b964f1494179c89f/clientconn_test.go#L811)

See https://pkg.go.dev/testing#hdr-Subtests_and_Sub_benchmarks for more details

Reply:
I'm not entirely sure I understand what you're asking. I thought this was table-driven: https://github.com/grpc/grpc-go/pull/7471/files#diff-bac34975c1867d508c8aad68d2f029ec7935220afab764701a1c1f7d478be75eR190-R216

I changed it to call `t.Run` inside the loop, if that's what you mean?

Reply:
Actually conventional way is to loop through flat list of test cases. Its okay if you have repetitions for some fields https://google.github.io/styleguide/go/decisions#table-driven-tests

Reply:
Ah, ok. Changed.